lib/peridiod/release/server.ex

```diff
@@ -509,7 +519,6 @@ defmodule Peridiod.Release.Server do
       end)
 
     case trusted_result do
-      {[], []} -> false
```

Comment:
this ensure that even if binaries_metadata got filtered to empty because everything was installed, that we don't treat that as untrusted